src/patternfly/components/AboutModalBox/about-modal-box-hero-image.hbs

```diff
@@ -0,0 +1,5 @@
+<img class="pf-c-about-modal-box__hero-image{{#if about-modal-box-hero-image--modifier}} {{about-modal-box-hero-image--modifier}}{{/if}}"
+  {{#if about-modal-box-hero-image--attribute}}
+    {{{about-modal-box-hero-image--attribute}}}
+  {{/if}}
+  {{> @partial-block}}/>
```

Comment:
remove the `/` - self-closing tags don't exist in HTML5, so no need for any closing syntax.

Reply:
Also, sorry I didn't catch this before, but you can remove `{{> @partial-block}}` from any `<img>` tag. If you really wanted to leave it in here, you could move it outside of the `<img>` tag, so that the `{{> @partial-block}}` would be displayed _after_ the image, but I don't think that's necessary.

Reply:
[avatar.hbs](https://github.com/patternfly/patternfly-next/pull/757/files#diff-0b64f6dc4c8dcdf44e94e01ab18d0bb4) is a good example of how it should be configured.

Reply:
fixed